python/zbar_init/__init__.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# _*_coding:utf-8_*_
+"""
+@Time   :  2021/2/5 19:14
+@Author :  wqh
+@Email  :  597935261@qq.com
+"""
+import os
+import sys
+from ctypes import CDLL
+
+SYS_MAP = {"msys": ["msys-zbar", ".dll"],
+           "win32": ["libzbar", ".dll"],
```

Comment:
Hmm... shouldn't this be written on an OS-independent way?

Reply:
yes . It provides a wrapper for uses on different OS to load ZBARlib anywhere not only the SYS PATH

Reply:
yes . It provides a wrapper for uses on different OS to load ZBARlib anywhere not only the SYS PATH

Reply:
yes . It provides a wrapper for uses on different OS to load ZBARlib anywhere not only the SYS PATH